Makefile

```diff
@@ -47,6 +47,14 @@ lint-go:
 .PHONY: lint-tf
 lint-tf: tfproviderlint tfproviderlintx
 
+.PHONY: lint-golangci
+lint-golangci:
```

Comment:
I think it should be safe to replace `lint-go` target with this one, since `golangci-lint` should be calling `gofmt` anyways IIRC

Reply:
done